configs/opencv-ovms/gst_capi/Dockerfile.ovms-capi-gst

```diff
@@ -82,6 +82,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt install -y jq -y && \
 ENV LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:/usr/local/lib/x86_64-linux-gnu/:/usr/local/lib:/ovms/lib:$LD_LIBRARY_PATH
 COPY gst_capi/pipelines /home/intel/gst-ovms/pipelines
 COPY gst_capi/launch-pipeline.sh /home/intel/gst-ovms/launch-pipeline.sh
+COPY models/2022/face-detection-retail-0005 /home/intel/gst-ovms/models/face-detection-retail-0005
```

Comment:
remove this, this model maynot be available before the first time you run the run.sh

Reply:
don't worry about the capi don't have model during the build time, the model will be available on the run time through the docker volume mount